features/upgrade/sdn/policy-upgrade.feature

```diff
@@ -78,3 +78,125 @@ Feature: SDN compoment upgrade testing
       | curl | -s | --connect-timeout | 5 | <%= cb.pod2ip %>:8080 |
     Then the step should fail
     And the output should not contain "Hello"
+
+
+  # @author asood@redhat.com
+  @admin
+  @upgrade-prepare
+  Scenario: Check the namespace networkpolicy for an application works well after upgrade - prepare
+    Given I switch to cluster admin pseudo user
+    When I run the :new_project client command with:
```

Comment:
@asood-rh do we have to use hardcoded project names? I think you can use the step `Given I have a project` and then save the project name into a clipboard.  This way, the project names are random to lessen the chance of name collision.

Reply:
@pruan-rht this is upgrade test, so we do want a known project, policy and application so that post upgrade we can check for the specific resources.

Reply:
@asood-rh oh ok, didn't catch that...looks good then

Reply:
/lgtm